bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -352,7 +360,7 @@ spec:
   - baremetal
   links:
   - name: Fence Agents Remediation
-    url: https://medik8s.io
+    url: https://medik8s.io/fence-agents-remediation/fence-agents-remediation
```

Comment:
Since this is broken I'd probably wait to merge docs PR first 
https://github.com/medik8s/docs/pull/63